rethinkdb/tests/data/tls/README.md

```diff
@@ -0,0 +1,21 @@
+# TLS certificates
+
+TLS certificates were generated using [`trustme-cli`](https://github.com/sethmlarson/trustme-cli):
```

Comment:
Nice for testing TLS 👍

Reply:
Indeed! I use that tool anytime I need to setup TLS on `localhost`. :-)